tools/ea/Ea.hs

```diff
@@ -170,7 +166,7 @@ genPayloadModule v txFiles = do
 
     let logger = genericLogger Warn TIO.putStrLn
 
-    payloadWO <- initPactService' Testnet00 (unsafeChainId 0) logger noSPVSupport $
+    payloadWO <- initPactService' Testnet00 (someChainId Testnet00) logger noSPVSupport $
```

Comment:
Hm I wonder when `Testnet00` snuck in here, since the `ChainwebVersion` is specifiable on the command line.

Reply:
Doubt it snuck in, my initial payload gen was hardcoded to Testnet00.

In truth we'll be generating payloads uniquely for each chain at some point, so this code will evolve.